ui/app/src/components/LegacyFormDialog/utils.ts

```diff
@@ -16,6 +16,7 @@
 
 import { PropsWithChildren } from 'react';
 import StandardAction from '../../models/StandardAction';
+import { DialogProps } from '@mui/material/Dialog';
```

Comment:
```suggestion
import { type DialogProps } from '@mui/material/Dialog';
```

Check other type imports.